examples/playwright/playwright.config.ts

```diff
@@ -1,5 +1,5 @@
 // *****************************************************************************
-// Copyright (C) 2021 logi.cals GmbH, EclipseSource and others.
+// Copyright (C) 2021-2023 logi.cals GmbH, EclipseSource and others.
```

Comment:
Just FYI, it's not wrong to update the Copyright date, but as per EF guidance, we consider that `git` is the source of truth, and so generally do not require this be done when an existing file is modified.